cirq/study/trial_result.py

```diff
@@ -66,15 +66,34 @@ def _big_endian_int(bits: Iterable[Any]) -> int:
     return result
 
 
+def _big_endian_bits(val: int, bit_count: int) -> List[bool]:
+    """Returns the big-endian bits of an integer.
+
+    For example, 18 becomes [True, False, False, True, False] when `bit_count`
+    is five.
+
+    Args:
+        val: The integer to get bits from.
+        bit_count: The number of desired bits in the result.
+
+    Returns:
+        The bits.
+    """
+    return [bool(val & (1 << i)) for i in range(bit_count)[::-1]]
+
+
```

Comment:
This should be a utility method that works with arbitrary `qid_shape`.

Reply:
Done. Extracted global utility methods taking a "base" argument that can be a list or an int.